{{cookiecutter.project_slug}}/Pipfile

```diff
@@ -30,3 +30,4 @@ pyyaml = "~=5.4.1"
 pillow = "~=8.2.0"
 pytest = "==6.2.4"
 factory-boy = "==3.2.0"
+cookiecutter= "1.6.0"
```

Comment:
Cookiecutter shouldn't be needed here. It's not used after the project is generated.